src/main/java/edu/harvard/iq/dataverse/api/Search.java

```diff
@@ -268,6 +284,28 @@ public Response search(
         }
     }
 
+    
+    @GET
+    @Path("/services")
+    public Response getSearchEngines() {
+        Map<String, SearchService> availableEngines = searchServiceFactory.getAvailableServices();
+        String defaultServiceName = JvmSettings.DEFAULT_SEARCH_SERVICE.lookupOptional().orElse("solr");
```

Comment:
Instead of hard-coding "solr" here, should we use the new INTERNAL_SOLR_SERVICE_NAME variable? We'd have to make it public.